packages/trimerge-sync/src/testLib/MemoryRemote.ts

```diff
@@ -15,22 +14,48 @@ import { PromiseQueue } from '../lib/PromiseQueue';
 export class MemoryRemote<CommitMetadata, Delta, Presence>
   implements Remote<CommitMetadata, Delta, Presence>
 {
+  active = false;
   private readonly remoteQueue = new PromiseQueue();
   private closed = false;
+  private onEvent: OnRemoteEventFn<CommitMetadata, Delta, Presence> | undefined;
 
   constructor(
     private readonly store: MemoryStore<CommitMetadata, Delta, Presence>,
     private readonly userId: string,
     private readonly clientStoreId: string,
-    { lastSyncCursor }: RemoteSyncInfo,
-    private readonly onEvent: OnRemoteEventFn<CommitMetadata, Delta, Presence>,
-  ) {
+  ) {}
+
+  private emit(event: SyncEvent<CommitMetadata, Delta, Presence>): void {
```

Comment:
Do not having the same buffer paradigm here expose tests to dropped events? I wouldn't want tests to have ordering requirements that are not identical to real world patterns.

Reply:
That's fair. Will add buffering to this.